motioneye/static/js/main.js

```diff
@@ -4735,41 +4745,41 @@ function addCameraFrameUi(cameraConfig) {
-                                '<div class="button icon camera-action-button mouse-effect light-on" title="turn light on"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect light-off" title="turn light off"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect alarm-on" title="turn alarm on"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect alarm-off" title="turn alarm off"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect snapshot" title="' + i18n.gettext("preni instantaron") + '"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect record-start" title="toggle continuous recording mode"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect up" title="up"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect down" title="down"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect left" title="left"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect right" title="right"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect zoom-in" title="zoom in"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect zoom-out" title="zoom out"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect preset preset1" title="preset 1"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect preset preset2" title="preset 2"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect preset preset3" title="preset 3"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect preset preset4" title="preset 4"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect preset preset5" title="preset 5"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect preset preset6" title="preset 6"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect preset preset7" title="preset 7"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect preset preset8" title="preset 8"></div>' +
-                                '<div class="button icon camera-action-button mouse-effect preset preset9" title="preset 9"></div>' +
+                                '<div class="camera-action-button-wrapper"><div class="button icon camera-action-button mouse-effect lock" title="lock"></div></div>' +
```

Comment:
I think it should be possible to skip the `camera-action-button-wrapper` div, isn't it? Or what is the purpose of centering a single div within its own parent div?